### ISSUE TITLE
How to draw Figure 9 and Figure 10 in your paper

### ISSUE THREAD

**Issue by IceStreams** [qichaoliu/CNN_Enhanced_GCN#6]
Hello, thank you very much for your hard work, I think it is very meaningful.
How do you draw Figure 9 (Visualization of graphs on different data sets) and Figure 10(a) (Visualization of feature fusion) in your paper. I think this representation is very intuitive.Could you share it, thank you very much!

**Comment by qichaoliu** [comment]
Thanks. 
How to draw Fig. 9: 
1. Compute spatial coordinate (x, y) of each superpixel by averaging all the coordinates of pixels in this superpixel. The average coordinate is taken as the position of its corresponding vertex (graph node) in figure.
2. Save all the edge weights and colour them in figure according to the coordinates of nodes computed in (1).

How to draw Fig.10:
1. Save the feature maps produced by CNN and GCN branches.
2. Use a classifier like logistic regression to classify these feature maps in (1) according to training labels. Then visualize the predicted probabilities generated by this classifier.

**Comment by IceStreams** [comment]
Thanks for your reply, about the fig9, how to colour the edge weights in figure?  What method did you use to draw it? Is it the matplotlib library?

**Comment by qichaoliu** [comment]
Yes, matplotlib. If there is an edge between two nodes (coordinates), just draw a line between them, and set its colour!
